documenting/automation/solution-pack/SP-Doc-Automation/setup.py

```diff
@@ -66,9 +67,27 @@ def __create_connector_data(self):
                 'Solution Pack does not contain data.json file under connector folder path: '.format(self.sp_dir_path + '/connectors'))
             return
 
+        connector_doc_data = self.__get_fortisoar_connector_doc_data()
+        logging.debug('Successfully fetched Fortinet FortiSOAR doc link')
         connector_data_path = self.sp_dir_path + '/connectors/data.json'
         connectors_data_json = json.load(open(connector_data_path))
         for connector in connectors_data_json:
             connectors_content += setup_connectors_data.substitute(connector_name=connector['label'],
-                                                                   connector_description=connector['description'], connector_doc_link='https://docs.fortinet.com/fortisoar/connectors') + '\n'
+                                                                   connector_description=connector['description'], connector_doc_link='https://docs.fortinet.com/fortisoar/connectors/{0}'.format(self.__get_connector_doc_link(connector_doc_data, connector['label']))) + '\n'
         return connectors_content
+
+    def __get_fortisoar_connector_doc_data(self):
+        response = requests.get(FORTISOAR_CONNECTOR_DOC_LINK)
+        return json.loads(response.text)
+
+    def __get_connector_doc_link(self, connector_doc_data, connector_label):
+        title_to_class = {item['title']: item['class']
+                          for item in connector_doc_data.get('connectors')}
+        if connector_label not in title_to_class.keys():
+            logging.debug(
+                'Doc link for connector - \"{0}\" has not been found.'.format(connector_label))
```

Comment:
should be info or error, no?
Also, can we add more details about why its not found?

Reply:
Should be info.
Not found if connector is of version 1.0.0 and docs and been not published yet.